prog/pwm/fancontrol.8

```diff
@@ -1,15 +1,20 @@
-.TH FANCONTROL 8 "September 2009" "lm-sensors 3"
+.TH FANCONTROL 8 "January 2018" "lm-sensors 3"
 .SH NAME
-fancontrol \- automated software based fan speed regulation
+fancontrol \- automated software based fan speed regulation to lower fan noise
```

Comment:
Not really. I don't think this needs to change.

Reply:
But lowering noise that is the core intention of the script, regulating a fan is nothing useful itself, right?

Reply:
As mentioned below, then you could simply set the fan speed to minimum. Another reason is to maximize system lifetime (fans last longer when running at slow speed, CPU lifetime increases if kept at low temperatures). It could also be to maximize system performance (while keeping noise level acceptable). I'd rather not make a guess as for _why_ one wants to control fan speed, or what the ultimate goal is. The script supports automated fan speed regulation, and it is up to the user to decide if the goal is to reduce noise or to optimize system lifetime or ...

Reply:
Well, i just never thought fancontrol as an alternative to "just reduce fan speed". I looked at it as alternative for full speed operation. Although i think turning off fans is a crazy alternative, you are right : it is a limitation of my view.

Reply:
The (intel-) guideline for the 4-wire interface for developers names the invention of PWM fan control  
"_a  means to reduce the overall system acoustics_". 
http://www.formfactors.org/developer/specs/4_Wire_PWM_Spec.pdf (page 7)

Reply:
One person's or group's limited view of the world doesn't make that limited view a reality.
Really, I am not getting paid to do this work, and simply don't have time to continue this bike-shedding. I'll get back to this series once I have time, and pick whatever I think makes sense. Please understand that I will not pick personal-view type changes.